WcaOnRails/app/views/results/_index_form.html.erb

```diff
@@ -0,0 +1,120 @@
+                        toggle: "tooltip",
+                        placement: "top"
+                      },
+                      title: event.name %>
+      </span>
+    <% end %>
+  </div>
+</div>
+
+<div class="form-group">
+  <%= label_tag(t("results.index.form_elements.region")) %>
+  <%= select_tag(:region, region_option_tags(selected_id: @region_param, real_only: true, use_world: true), class: "form-control") %>
+</div>
+
+<br>
+
+<% unless @event_param == "333mbf" %>
+  <div class="form-group">
+    <%= label_tag(t("results.index.form_elements.type_selector.type")) %>
+    <div id="type" class="btn-group">
+      <% @types.each do |type| %>
+        <%= link_to t("results.index.form_elements.type_selector.#{type}"),
+                      request.params.merge(type: type),
+                      class: "btn btn-primary" + (type == @type_param ? " active" : "") %>
+      <% end %>
+    </div>
+  </div>
+<% end %>
+
+<div class="form-group">
+  <%= label_tag(t("results.index.form_elements.years_selector.years")) %>
+  <div id="years" class="btn-group">
+    <%= link_to t("results.index.form_elements.years_selector.all_years"), request.params.merge(years: "all years"), class: "btn btn-info" + (@is_all_years ? " active" : "") %>
+    <div class="btn-group">
+      <% only_active = @is_only %>
+      <ul class="dropdown-menu years-only">
+        <% @years.each do |year| %>
+          <li class="only <%= (only_active && year.to_s == @year ? " active" : "") %>"><%= link_to year.to_s, request.params.merge(years: "only #{year}") %></li>
+        <% end %>
+      </ul>
+      <label id="only-years" class="btn btn-info <%= only_active ? " active" : "" %>" data-toggle="dropdown">
+        <span class="caption">
+          <% if only_active %>
+            <%= t("results.index.form_elements.years_selector.only_html", year: @year) %>
+          <% else %>
+            <%= t("results.index.form_elements.years_selector.only") %>
+          <% end %>
+        </span>
+      </label>
+    </div>
+    <div class="btn-group">
+      <% until_active = @is_until %>
+      <ul class="dropdown-menu years-until">
+        <% @years.each do |year| %>
+          <li class="until <%= (until_active && year.to_s == @year ? " active" : "") %>"><%= link_to year.to_s, request.params.merge(years: "until #{year}") %></li>
+        <% end %>
+      </ul>
+      <label id="until-years" class="btn btn-info <%= until_active ? " active" : "" %>" data-toggle="dropdown">
+        <span class="caption">
+          <% if until_active %>
+            <%= t("results.index.form_elements.years_selector.until_html", year: @year) %>
+          <% else %>
+            <%= t("results.index.form_elements.years_selector.until") %>
+          <% end %>
+        </span>
+      </label>
+    </div>
+  </div>
+</div>
+
+<div class="form-group">
+  <%= label_tag(t("results.index.form_elements.show_selector.show")) %>
+  <div id="show" class="btn-group">
+    <div class="btn-group">
```

Comment:
I think yes, as I could only have one dropdown menu per button group. I'm leaving this conversation as unresolved for now just in case someone has a better solution!